projects/halide/build.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash -eu
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################ 
+
+export LLVM_ROOT=$SRC/llvm-install
+export LLVM_CONFIG=$LLVM_ROOT/bin/llvm-config
+
+cmake -G Ninja  -S . -B build -DCMAKE_BUILD_TYPE=Release \
```

Comment:
Feel free to contribute an `ossfuzz` preset to Halide's CMakePresets.json fils

Reply:
I'll look into it 👍